.github/workflows/nightly.yml

```diff
@@ -14,6 +14,46 @@ jobs:
       browser-target: chrome@latest
     secrets: inherit
 
+  track-size:
+    name: Track Build Size
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: main
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22.11.0 # See package.json for the stable node version that works with our testing.  Do not change this unless you know what you are doing as some node versions do not play nicely with our testing server.
+      - name: Install project dependencies
+        run: npm ci
+      - name: Report lite size to NR
+        uses: newrelic/capture-build-size@main
+        with:
+          nr-api-key: ${{ secrets.INTERNAL_STAGING_INGEST_LICENSE_KEY }}
+          nr-account-id: ${{ secrets.INTERNAL_STAGING_INGEST_ACCOUNT_ID }}
+          nr-env: 'staging'
+          analysis-file-url: 'https://js-agent.newrelic.com/dev/nr-rum-standard.stats.json'
```

Comment:
Didn't make immediate sense to me why this grabs the stat file from CDN when there wasn't a build & publish step preceding this. While nightly runs on `main` which should be updated by publish-dev, I think this is either unclear or redundant entirely. Put another way, since publish-dev runs on all `main` commits and does this already, we don't need nightly to do this.